### ISSUE TITLE
[Property Request]: Cal-ITP services

### ISSUE THREAD

**Issue by AnthonyRollins** [cal-itp/customer-success#405]
### Name of Custom Property

Cal-ITP Services

### What is the problem that this is solving?

Users of Hubspot need a way to showcase which Cal-ITP services an agency is a part of. Currently, you have to view each pipeline they are associated with to understand what services they have. This new property would allow a high-level view of agencies attached to one, two, or even three Cal-ITP services.

### Object Association

Companies

### If you chose Company in the question above, which company type does this information apply to?

- [X] Transit Agency
- [ ] Vendor
- [ ] N/A
- [ ] Unsure

### If you chose Deal in the question above, which pipeline does this information apply to?

- [ ] Payments Pipeline
- [ ] Contract Outreach
- [ ] MDIP Outreach
- [ ] GRaaS
- [ ] Cubic/Umo API Key Outreach
- [ ] Cal-ITP Benefits
- [ ] Software for Scheduling
- [ ] Payments Dashboard
- [ ] Relevant Pipeline has not been created yet
- [X] N/A
- [ ] Unsure

### Who is responsible for updating and maintaining this property?

Customer Success team will update this pipeline

### Updating Cadence

Quarterly

### When does this property need to be implemented?

2/15/24

### Not sure what you need?

No

**Comment by AnthonyRollins** [comment]
Brainstorm [spreadsheet](https://docs.google.com/spreadsheets/d/1fhGv9ZwARW5IXSEXXWsolwihigNWyuSM1D2Fsqnf58E/edit#gid=0) with all the potenial services that could be added to agencies.

**Comment by AnthonyRollins** [comment]
@o-ram @laneymangan I [compiled all services](https://docs.google.com/spreadsheets/d/1fhGv9ZwARW5IXSEXXWsolwihigNWyuSM1D2Fsqnf58E/edit#gid=0) that we can add to companies, let me know if there is anything missing from this list.

**Comment by laneymangan** [comment]
Perhaps we add something for the Scheduling(Remix) work. Not exactly sure what the details will be moving forward but for now we could list whether they took part in the pilot.

We could also add something for the Farebox work, but there's not many agencies involved so not totally sure if it's necessary? @o-ram

**Comment by AnthonyRollins** [comment]
@laneymangan Do we have a list of the scheduling work? I can add the farebox work, too. The number of agencies isn't an issue, as we can always remove them.

**Comment by laneymangan** [comment]
@AnthonyRollins For scheduling it's Simi Valley, VCTC, SBCAG/CAE, and I believe Merced (is that right about merced @o-ram?) For the farebox work I only know Glenn Transit but I'll check with Angelly on the other two

**Comment by laneymangan** [comment]
Actually, we had it in our notes--the farebox cohort is Glenn Transit, MST, and VCTC

**Comment by AnthonyRollins** [comment]
I've updated my spreadsheet with Farebox and scheduling.

**Comment by AnthonyRollins** [comment]
@o-ram Review the [spreadsheet](https://docs.google.com/spreadsheets/d/1fhGv9ZwARW5IXSEXXWsolwihigNWyuSM1D2Fsqnf58E/edit#gid=0) before import

**Comment by o-ram** [comment]
Reviewed and made a few minor content updates (the structure overall makes sense):
- Updated the MDIP list based on the list of CA agencies who are co-authors or consigners
- Updated ODS - Nevada does not have it (I'm their AM and they've asked about it but haven't confirmed that they are using it).
- Made sure the payments list matched the current pipeline

One question I have: 
- Do we want to include anything about GTFS in here? I know that always becomes a loaded question, but that is a "service" of sorts. Perhaps we can start with data creation? That's one area of GTFS assistance that we have tracked in HubSpot and is pretty easy to label. Let me know what you both think, and I can easily add that to the spreadsheet if desired. Other than that, this looks good for importing!

Question on maintenance expectations (not intending it to block importing but don't want to forget - fine for these to be moved to be discussed elsewhere):

- Are we intending these to all be manual updates that AMs make? Or are we going to implement workflows where we can make it so that when a deal makes it to a certain pipeline stage, the associated company has its service property filled out?

**Comment by AnthonyRollins** [comment]
Yes, let's add the GTFS data creation. I think that is specific enough to be included in this list. 

As for maintenance, this is something the AM can update in the meantime, but I would like us to get to the point where another action, moving to a stage, triggers this property to change. There are some low-hangings that we can make sure we manually update, for instance, ODS, and we will have to figure out the best process in the future for some of the more complicated ones like GTFS and Payments.

**Comment by o-ram** [comment]
@AnthonyRollins the link to the data creation view [is here](https://app.hubspot.com/contacts/5519226/objects/0-5/views/35011489/board), but I believe I pulled all the agencies who meet the threshold out and put them in the first tab of the [spreadsheet](https://docs.google.com/spreadsheets/d/1fhGv9ZwARW5IXSEXXWsolwihigNWyuSM1D2Fsqnf58E/edit?gid=0#gid=0). I'll need to check with the other AMs next week though to make sure I didn't miss anyone.

**Comment by AnthonyRollins** [comment]
@o-ram, Could you look at the [final import spreadsheet](https://docs.google.com/spreadsheets/d/1fhGv9ZwARW5IXSEXXWsolwihigNWyuSM1D2Fsqnf58E/edit?gid=382874256#gid=382874256) and make sure I got the right domain and companies?

**Comment by o-ram** [comment]
@AnthonyRollins I reviewed it! All companies and domains look good. While I was in there, I went ahead and updated the payments list since that has grown since we first drafted this. That is now current as well. From my pov, this is ready to import!

**Comment by AnthonyRollins** [comment]
I added the new property to Hubspot and added the property to the Transit agency section.
